### PR TITLE
Umbraco Commerce Taxes Documentation

### PR DESCRIPTION
**Pull request by mattbrailsford**
## Description

Umbraco Commerce Taxes Documentation, including docs for Sales Tax Providers and the new Tax Calculation Methods.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco Commerce 14.1.0

## Deadline (if relevant)

TBD

### CODE REVIEW

File: 14/umbraco-commerce/reference/taxes/calculated-tax-rates.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+﻿---
+description: Calculated Rate Taxes in Umbraco Commerce.
+---
+
+# Calculated Tax Rates
+
+Fixed rate taxes allows you to dynamically calculate the tax obligations of an order by using third party calculation platforms. This is a more complex option that is useful for countries that have different tax rates for different types of products or for different regions within the country (the norm in the US).
```
Comment (sofietoft) [inline 14/umbraco-commerce/reference/taxes/calculated-tax-rates.md]:
```suggestion
Fixed-rate taxes allow you to dynamically calculate the tax obligations of an order by using third-party calculation platforms. This complex option is useful for countries with different tax rates for different product types or regions within the country (the norm in the US).
```

File: 14/umbraco-commerce/reference/taxes/fixed-tax-rates.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+﻿---
+description: Fixed Rate Taxes in Umbraco Commerce.
+---
+
+# Fixed Tax Rates
+
+Fixed rate taxes allows you to define a single fixed tax rate that will be applied for all products of the same type shipped to the same country. This is a simple option that is useful for countries that have a fixed tax rate (the norm in EU countries).
```
Comment (sofietoft) [inline 14/umbraco-commerce/reference/taxes/fixed-tax-rates.md:7]:
```suggestion
Fixed-rate taxes allow you to dynamically calculate the tax obligations of an order by using third-party calculation platforms. This complex option is useful for countries with different tax rates for different product types or regions within the country (the norm in the US).
```

File: 14/umbraco-commerce/SUMMARY.md [hunk 3]
```diff
@@ -58,6 +58,7 @@
   * [Complex Variants](key-concepts/product-variants/complex-variants.md)
 * [Properties](key-concepts/properties.md)
 * [ReadOnly and Writable Entities](key-concepts/readonly-and-writable-entities.md)
+* [Sales Tax Providers](./key-concepts/sales-tax-providers.md)
```
Comment (sofietoft) [inline 14/umbraco-commerce/SUMMARY.md:61]:
```suggestion
* [Sales Tax Providers](key-concepts/sales-tax-providers.md)
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
I've added some suggestions here and there.
Also took the liberty to make some changes directly - mainly to the lists, where we prefer having the images on their own line and not directly as part of the list.

**Comment by mattbrailsford** [comment]
All looks good to me 👍

**Review by sofietoft — APPROVED** [review]

**Pull request merged**